ci/test/00_setup_env_native_nowallet.sh

```diff
@@ -8,7 +8,7 @@ export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_nowallet
 export DOCKER_NAME_TAG=ubuntu:18.04  # Use bionic to have one config run the tests in python3.6, see doc/dependencies.md
-export PACKAGES="python3-zmq clang-5.0 llvm-5.0"  # Use clang-5 to test C++17 compatibility, see doc/dependencies.md
+export PACKAGES="python3-zmq clang-7 llvm-7 libstdc++-8-dev"  # Use clang-7 to test C++17 compatibility, see doc/dependencies.md
```

Comment:
```suggestion
export PACKAGES="python3-zmq clang-7 llvm-7 libc++abi-7-dev libc++-7-dev"  # Use clang-7 (with libc++-7) to test C++17 compatibility, see doc/dependencies.md
```

Reply:
> Is there a reason in ci to check libstdc++-8 twice

It's not quite twice, given one build is Clang and one GCC. I had wanted this so that the Clang & libstdc++ case was tested, given that is going to be the most likely case when developers / users are compiling using (older) Clang. However happy to change this to use libc++ instead.

Reply:
Currently no test coverage exists for libc++, so CI can't catch "bugs" like #22342.

Was there ever a time when clang&libstdc++ was broken, but gcc and clang&libc++ both worked?

Reply:
> so CI can't catch "bugs" like #22342.

Yea for this case it's worth using libc++.

Reply:
The wallet code is disabled, so it can't catch *this* specific instance, but in consensus code it should be able to catch.

Reply:
Sure, I'm just talking about introducing code that will break the libc++ 7 requirement.